site/content/en/docs/manual/advanced/annotation-with-brushing-tools.md

```diff
@@ -0,0 +1,121 @@
+---
+title: 'Annotation with brushing tools'
+linkTitle: 'Annotation with brushing tools'
+weight: 13
+description: 'Guide to annotating tasks using  brushing tools.'
+---
+
+The brushing tool creates a segmentation mask (further - mask) on the top of the image. Use it, when you need to add a label to an object that has several separated parts, but must be counted as one, for example - a house hiding behind trees or a pillar behind a traffic sign.
+
+You can annotate images manually or upload already annotated images, you can also convert polygon annotation to masks and the other way around.
```

Comment:
> can also convert polygon annotation to masks

How is it possible? I only can see one way to do that, is to export dataset in one of segmentation masks format and import it back with disabled option 'Convert masks to polygons'. The tool itself does not provide such conversion capabilities on UI.

Reply:
This paragraph seems extra here.